### ISSUE TITLE
CI smoke test for critical networking endpoints

### ISSUE THREAD

**Issue by enrichman** [epinio/epinio#2038]
Some endpoints are using websockets and silently broke in different Kubernetes version.
We should add at least a couple of smoke test with different version to check that they are still working with newer/older versions.

Endpoints to check:
- port-forward
- exec

Version: at least a couple (1.20 and 1.24)

**Comment by mmartin24** [comment]
Work done until this moment:

**CI**:
- Created new workflow provisionally named `custom-ci` 
- Added new dispatch `k3s_version`. 
- The entry passes to `acceptance-cluster-setup.sh` script through env var `K3S_CUSTOM` added to the main script. However it does not work with make so I substituted it by calling the script directly like:
```
scripts/acceptance-cluster-setup.sh $K3S_CUSTOM
```
- Changed script  `acceptance-cluster-setup.sh` to use custom-ci only when passed and default if not.
I checked its functioning in self hosted runner and seems to work ok: https://github.com/mmartin24/epinio_fork/actions/runs/4158691011

**Tests**:
- Added new makefile line `test-acceptance-api-critical-endpoints` with momentary label `criticalep` targetting desired tests.

---
Problems found so far:
- `make prepare_environment_k3d` seems not to work after. Error thrown:
```
https://cert-manager.io/docs/usage/ingress/
fatal: No names found, cannot describe anything.
GOARCH="amd64" GOOS="linux" CGO_ENABLED=0 go build  -ldflags '-X github.com/epinio/epinio/internal/version.Version=' -o dist/epinio-linux-amd64
fatal: No names found, cannot describe anything.
fatal: No names found, cannot describe anything.
fatal: No names found, cannot describe anything.
make: *** [Makefile:76: build-images] Error 128
```

- When executing `test-acceptance-api-critical-endpoints` it throws: 
```
make: *** [Makefile:121: test-acceptance-api-critical-endpoints] Error 1.
```
So far it seems related somehow to flag `--poll-progress-after` in Ginkgo execution. It seems to work when using `--slow-spec-threshold`. So this seems to work:

```
ginkgo -v --nodes 1 --slow-spec-threshold 200s --randomize-all --flake-attempts=1 --fail-on-pending --label-filter "criticalep"  acceptance/api/v1/.
```

I am going to pause this task for a moment given those 2 issues to attend other priorities and will come back to it later.
